lib/widgets/autocomplete.dart

```diff
@@ -123,23 +124,31 @@ class _ComposeAutocompleteState extends State<ComposeAutocomplete> with PerAccou
 
   Widget _buildItem(BuildContext _, int index) {
     final option = _resultsToDisplay[index];
+    Widget avatar;
```

Comment:
Added this as a non-nullable `Widget` and assigned it a `SizedBox.shrink` for the parts not yet implemented. Not sure if this approach is better, or making it a `Widget?` and not assigning anything for parts not yet implemented!!

Reply:
Hmm. Probably the right answer is actually that we should delete those two other result subclasses that we currently never use :-). That'd be good to do as an [NFC](https://github.com/zulip/zulip-mobile/blob/main/docs/glossary.md#nfc) prep commit.

The details of them will always be available in the Git history. Could also include like one line for each of them, commented out, with their respective TODO comments moved there.

Reply:
If these instead represented some situation that is already possible and that we just didn't have avatars for (or didn't yet), then one thing we'd want to be sure to do is to get the layout right — there should be a blank space with the right size for an avatar. So that might look like using `AvatarShape` unconditionally, with a child that's either `AvatarImage` or some placeholder, potentially `SizedBox.shrink()`.

If we used just `SizedBox.shrink`, it looks like that'd get the layout wrong, because it looks below like this is being used in a context that doesn't dictate its size and expects it to provide its own size. It wouldn't currently matter, because those cases are unreachable… but better not to rely on that, hence the prep commit mentioned above.